theme/scripts/cart.ts

```diff
@@ -200,7 +285,7 @@ export default(function() {
       const addToCartVariantId = addToCartButton.getAttribute(Cart.ATTRIBUTES.variantId);
 
       if (addToCartVariantId === variantId) {
-        if (action === Action.ADD) {
+        if (action === Action.ADD || action === Action.UPDATE) {
```

Comment:
If the cart line item has been changed, we still want to show "ADDED TO CART" instead of something like "CART UPDATED" I'm guessing.